CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.12.32+2
+
+* Provide a work around for dart-lang/sdk/issues/32556.
```

Comment:
This doesn't really communicate to downstream users what changed. Those users don't care what the SDK bug was, they care what user-visible thing was going wrong that is now fixed. Consider writing something like "Work around an SDK bug that caused timeouts in asynchronous code."

Also, for future reference, it's probably a good idea to use a link that users can follow rather than something like `dart-lang/sdk/issues/32556`.

Reply:
Done.